genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -442,11 +442,38 @@ lost and failed by the Genie leader
 |http
 |no
 
+|genie.tasks.database-cleanup.application-cleanup.skip
+|Skip the Applications table when performing database cleanup
+|false
+|yes
+
 |genie.tasks.database-cleanup.cluster-cleanup.skip
 |Skip the Clusters table when performing database cleanup
 |false
 |yes
 
+|genie.tasks.database-cleanup.command-cleanup.skip
+|Skip the Commands table when performing database cleanup
+|false
+|yes
+
+|genie.tasks.database-cleanup.command-deactivation.commandCreationThreshold
```

Comment:
Minor: this property is named `retention` for jobs. Shouldn't they be named the same?

Reply:
I thought about this when I was doing it. `retention` was a hard line in the sand where everything before time x was just going to be deleted so we were keeping a floating window of y days. In this case even if the command was created before this time it could be left unaffected as it's still used "recently" by a job. Since this is the deactivation setting and not the full delete setting I left it this way so I could distinguish it from the job threshold